torchci/lib/bot/autoLabelBot.ts

```diff
@@ -249,8 +314,55 @@ function myBot(app: Probot): void {
         break;
     }
 
-    if (newLabels.length) {
-      await addLabels(context, newLabels);
+    const newLabelsFromLabelToLabelConfig =
+      await getLabelsFromLabelToLabelConfig(
+        context,
+        labelToLabelConfigTracker,
+        labels,
+        label
+      );
+    for (const label of newLabelsFromLabelToLabelConfig) {
+      addLabel(labelSet, newLabels, label);
+    }
+
+    const filtered = await filterCIFlowLabels(true, newLabels);
+    if (filtered.length) {
+      await addLabels(context, filtered);
+    }
+  });
+
+  app.on("pull_request.labeled", async (context) => {
+    // Careful! Read the above comments about label addition
+
+    const label = context.payload.label!.name;
```

Comment:
Nit: Could we rename them to `addedLabel` and `existingLabels` like how you name them in getLabelsFromLabelToLabelConfig?  My mental map is lying to me by telling label is a member of labels lol

Reply:
Maybe doing the same for issues.labeled event too for consistency

Reply:
did this for pull request labeled, I want to refactor this file so I'll do the changes for other funcs in a different PR